Lib/test/test_typing.py

```diff
@@ -4722,6 +4722,21 @@ class D(Generic[T]): pass
         with self.assertRaises(TypeError):
             D[()]
 
+    def test_generic_init_subclass_not_called_error(self):
+        class Base:
+            def __init_subclass__(cls) -> None:
+                # Oops, I forgot super().__init_subclass__()!
+                pass
+
+        class Sub(Base, Generic[T]):
+            pass
+
+        msg = (r"type object '.*Sub' has no attribute " +
+              re.escape("'__parameters__'. Maybe you forgot to call "
+                        "super().__init_subclass__()?"))
```

Comment:
```suggestion
        msg = (
            r"type object '.*Sub' has no attribute '__parameters__'\. "
            r"Maybe you forgot to call super\(\)\.__init_subclass__\(\)\?"
        )
```

Actually the `re.escape` stuff is a bit more annoying. So we can do it manually (previously, what was hard to read was the with + msg, so having a separate variable for the message already improves readability).